src/cli-client/internals/cli/cli_main.go

```diff
@@ -0,0 +1,66 @@
+package cli
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/canonical/go-flags"
+	"github.com/canonical/oci-factory/src/cli-client/internals/logger"
+)
+
+type CmdRelease struct {
+	Revision int `long:"revision" description:"The revision number of the image release"`
+}
+
+const longDescription = `The OCI Factory CLI client is a tool that builds, tests, and releases
+the OCI images owned by Canonical using the Github workflow in 
+the OCI Factory repository.
+`
```

Comment:
nit: let's format this a bit
```suggestion
const longDescription = `
The OCI Factory CLI client is a tool that builds, tests, and releases the OCI
images owned by Canonical using the Github workflow in the OCI Factory repository.
`
```